README.md

```diff
@@ -56,3 +56,19 @@ the default.
 ## Docker image
 This repository builds a docker image that is then used as the executor for the `behave` Circle CI job.
 You are unlikely to need to use the image directly elsewhere.
+
```

Comment:
Should this be in the other readme for how to run the tests?

Reply:
I've moved it into the same README.md that contains all the instructions about how to use the tests in docker and run them. Including extra documentation on the commands to run, as I notice we haven't written that down anywhere.